### ISSUE TITLE
Difference between nn.Conv1d and nn.Linear

### ISSUE THREAD

**Issue by XuyangBai** [fxia22/pointnet.pytorch#39]
Hi, I think the nn.Conv1d and nn.Linear are the same for pointwise convolution( if `kernel_size = 1` for conv1d), so could you please provide some reasons about why you choose nn.Conv1d for the first three layers and choose nn.Linear for the last two layers ?

**Comment by fxia22** [comment]
There is no particular reason for using conv1d. I used conv1d because the original tensorflow implementation used it. I could also use linear layers.

**Comment by XuyangBai** [comment]
Thank you for your reply.

**Comment by AkashGanesan** [comment]
There's a difference.  Here's an explanation.  

https://stackoverflow.com/questions/55576314/conv1d-with-kernel-size-1-interpretation

**Comment by YannDubs** [comment]
@AkashGanesan There's no difference in the result besides intialization and speed (linear being quicker). See my answer in the stack overflow question you linked to:

https://stackoverflow.com/questions/55576314/conv1d-with-kernel-size-1-interpretation/56685503#56685503

**Comment by adosar** [comment]
@XuyangBai @fxia22 

> Hi, I think the nn.Conv1d and nn.Linear are the same for pointwise convolution( if `kernel_size = 1` for conv1d), so could you please provide some reasons about why you choose nn.Conv1d for the first three layers and choose nn.Linear for the last two layers ?

Note that since `BatchNorm1D` expects an input of shape `(N, C, L)`,  we  will need to reshape each time before and after the `Conv1D`.

**Comment by adosar** [comment]
@fxia22 I don't expect significant speed/performance differences, but could we set `bias=False` to all `Conv1d` layers since they are followed by `BatchNorm1d` (same of course goes for the `Linear` in the head MLP)?
